app/cosmo/base.toml

```diff
@@ -1126,7 +1130,7 @@ name = "Northwest"
 description = "Northwest temperature sensor"
 sensors = { temperature = 1 }
 removable = true
-refdes = "J47"
+refdes = ["J47", "U1"]
```

Comment:
Weak vibes: the temperature sensor board is basically just a temperature sensor on a stick, so I don't think it's necessary to have the chip listed separately.

Reply:
yeah, i could go either way on these, i think there was one board (possibly one of the devboards?) where it was `Jx_U1` and i wanted to be consistent with that, but I can change them all to just the jack's refdes.